ant-call-maven/build.xml

```diff
@@ -0,0 +1,28 @@
+<project name="ant-call-maven" basedir=".">
+  <description>
+    An Ant build that call maven
+  </description>
+
+  <macrodef name="mvn">
+    <attribute name="pom" default="pom.xml" />
+    <attribute name="dir" default="" />
+    <element name="maven-goals" implicit="true" description="Maven goals to execute" />
+    <sequential>
+
+      <property name="maven-wrapper"
+                location="${basedir}/mvnw"/>
+      <exec executable="${maven-wrapper}" dir="@{dir}">
+        <arg value="-f@{pom}" />
+        <maven-goals/>
```

Comment:
Should this also use `--batch-mode`?

Reply:
I'm not sure. If I understood our conversation correctly, `--batch-mode` will not create a build scan, but that is what I want. I tried to document the necessary modifications one needs to make to get this working in `README.md`. If your plan is to have an example that is copy-pasteable and does work out of the box I would prefer configuring ToS in `gradle-enterprise.xml`.

Reply:
`--batch-mode` will make the Maven execution non-interactive, i.e. it will not ask to ask to accept the terms of service when no server is configured. When a server is configured it will upload a build scan, though.

Reply:
Done